portability-core/src/main/java/org/dataportabilityproject/serviceProviders/google/GoogleStaticObjects.java

```diff
@@ -64,4 +63,9 @@ public static HttpTransport getHttpTransport() {
   public static FileDataStoreFactory getDataStoreFactory() {
     return DATA_STORE_FACTORY;
   }
+
+  /**
+   * Static values used in converters
```

Comment:
Maybe add a little more doc on why MAX_ATTENDEES is set at this value

Reply:
Adding a TODO to evaluate this.